patches/bytenode+1.3.7.dev.patch

```diff
@@ -0,0 +1,381 @@
+diff --git a/node_modules/bytenode/lib/compileFile.js b/node_modules/bytenode/lib/compileFile.js
+new file mode 100644
+index 0000000..314eeee
+--- /dev/null
++++ b/node_modules/bytenode/lib/compileFile.js
```

Comment:
This looks hard to maintain - is there a bug in the library or we just needed a specific feature?

It is possible to write our own `compileElectronCode`, wrapping the `bytenode` library functionality, and put it in the monorepo (eg under `scripts`)? That seems like it'd be easier to maintain, long term, than patching the package.